### DISCUSSION TITLE
Find more matches

### DISCUSSION THREAD

**Discussion by Gercov** [prisma/prisma#12558] [Q&A]
I'm trying made request, that will find closest by 5 fields: 
- name: string
- subscribers: number[]
- about?: string
- photo?; string
- verify: boolean

All fields may be changed, and i want find closest to old data. How made it?

**Comment by nurul3101** [comment]
Hey Gercov 👋 

Could you please elaborate on what you are trying to achieve? 
Maybe share your schema file and the query which you are trying to invoke?

**Reply by vladi-stevanovic** [reply]
👋 Hello @Gercov  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
